Description: Avoid crash during startup
 This patch allow compatibility with the new versions of
 glib2.0 removing gtk.gdk.threads_init() because this is now deprecated.
Forwarded: no
Author: Elías Alejandro Año Mendoza <ealmdz@gmail.com>
Last-Update: 2014-09-25

Index: radiotray/src/SysTray.py
===================================================================
--- radiotray.orig/src/SysTray.py	2014-09-25 18:15:12.000000000 -0500
+++ radiotray/src/SysTray.py	2014-09-25 18:21:49.000000000 -0500
@@ -46,6 +46,7 @@
 from AppIndicatorGui import AppIndicatorGui
 from TooltipManager import TooltipManager
 from Context import Context
+from glib import glib_version
 
 import dbus
 import textwrap
@@ -182,8 +183,9 @@
     
         
     def run(self):
-        gtk.gdk.threads_init()
-        gtk.main()
+		if glib_version	< (2, 41, 0):
+			gtk.gdk.threads_init()
+		gtk.main()
 
 
     
